platform-dependent-components/problem-solver/cxx/sections-module/agent/remove_section_agent.cpp

```diff
@@ -85,29 +85,28 @@ bool RemoveSectionAgent::RemoveSection(ScAddr const & section, ScAddr const & pa
 
 bool RemoveSectionAgent::RemoveSection(ScAddr const & section)
 {
-  SC_LOG_DEBUG("RemoveSectionAgent: section system idtf is " << m_memoryCtx.HelperGetSystemIdtf(section) << ".");
+  SC_LOG_DEBUG("RemoveSectionAgent: section system idtf is " << m_context.HelperGetSystemIdtf(section) << ".");
```

Comment:
now there are versions of this logs like SC_AGENT_LOG_DEBUG, SC_AGENT_LOG_ERROR, they append agent name to log so it can be not repeated in every message